LibGit2Sharp.Tests/GlobalSettingsFixture.cs

```diff
@@ -53,33 +53,33 @@ public void TryingToResetNativeLibraryPathAfterLoadedThrows()
             Assert.Throws<LibGit2SharpException>(() => { GlobalSettings.NativeLibraryPath = "C:/Foo"; });
         }
 
-        [SkippableTheory]
-        [InlineData("x86")]
-        [InlineData("x64")]
-        public void LoadFromSpecifiedPath(string architecture)
-        {
-            Skip.IfNot(Platform.IsRunningOnNetFramework(), ".NET Framework only test.");
+        //[SkippableTheory]
+        //[InlineData("x86")]
+        //[InlineData("x64")]
+        //public void LoadFromSpecifiedPath(string architecture)
+        //{
+        //    Skip.IfNot(Platform.IsRunningOnNetFramework(), ".NET Framework only test.");
 
-            var nativeDllFileName = NativeDllName.Name + ".dll";
-            var testDir = Path.GetDirectoryName(typeof(GlobalSettingsFixture).Assembly.Location);
-            var testAppExe = Path.Combine(testDir, $"NativeLibraryLoadTestApp.{architecture}.exe");
-            var tempDir = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
-            var platformDir = Path.Combine(tempDir, "plat");
+        //    var nativeDllFileName = NativeDllName.Name + ".dll";
+        //    var testDir = Path.GetDirectoryName(typeof(GlobalSettingsFixture).Assembly.Location);
+        //    var testAppExe = Path.Combine(testDir, $"NativeLibraryLoadTestApp.{architecture}.exe");
+        //    var tempDir = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+        //    var platformDir = Path.Combine(tempDir, "plat");
 
-            try
-            {
-                Directory.CreateDirectory(Path.Combine(platformDir, architecture));
-                File.Copy(Path.Combine(GlobalSettings.NativeLibraryPath, architecture, nativeDllFileName), Path.Combine(platformDir, architecture, nativeDllFileName));
+        //    try
+        //    {
+        //        Directory.CreateDirectory(Path.Combine(platformDir, architecture));
+        //        File.Copy(Path.Combine(GlobalSettings.NativeLibraryPath, architecture, nativeDllFileName), Path.Combine(platformDir, architecture, nativeDllFileName));
 
-                var (output, exitCode) = ProcessHelper.RunProcess(testAppExe, arguments: $@"{NativeDllName.Name} ""{platformDir}""", workingDirectory: tempDir);
+        //        var (output, exitCode) = ProcessHelper.RunProcess(testAppExe, arguments: $@"{NativeDllName.Name} ""{platformDir}""", workingDirectory: tempDir);
 
-                Assert.Empty(output);
-                Assert.Equal(0, exitCode);
-            }
-            finally
-            {
-                DirectoryHelper.DeleteDirectory(tempDir);
-            }
-        }
+        //        Assert.Empty(output);
+        //        Assert.Equal(0, exitCode);
+        //    }
+        //    finally
+        //    {
+        //        DirectoryHelper.DeleteDirectory(tempDir);
+        //    }
+        //}
```

Comment:
🤔 If we want to address it later, let's create an issue in the repo so we don't forget about it.

Reply:
https://github.com/mendix/libgit2sharp/issues/2#issue-702650660

issue created